.azure/app-cloud-e2e.yml

```
@@ -86,7 +86,10 @@ jobs:
         path: $(pip_cache_dir)
       displayName: Cache pip
 
-    - bash: python -m pip install -r requirements/app/devel.txt --quiet --find-links ${TORCH_URL}
+    - bash: python -m pip install -e . --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
+      displayName: 'Install lightning'
+
+    - bash: python -m pip install -r requirements/app/test.txt -r requirements/app/ui.txt --find-links ${TORCH_URL}
       env:
         TORCH_URL: https://download.pytorch.org/whl/cpu/torch_stable.html
```

nit: the torch URL seems unnecessary here.
```suggestion
    - bash: python -m pip install -r requirements/app/test.txt -r requirements/app/ui.txt
```